activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -894,4 +902,60 @@ public void should_not_returnProcessesToNonInitiatorUser() {
         assertThat(processInstancePage).isNotNull();
         assertThat(processInstancePage.getContent()).isEmpty();
     }
+
+    @Test
+    public void should_returnProcessToTaskAssignee() {
+        //given
+        ProcessInstance processInstance = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(SINGLE_TASK_PROCESS)
+            .build());
+
+        securityUtil.logInAs("garth");
+
+        //when
+        processInstance = processRuntime.processInstance(processInstance.getId());
+
+        assertThat(processInstance.getInitiator()).isEqualTo("user");
+    }
+
+    @Test
+    public void should_returnProcessToTaskCandidate() {
```

Comment:
maybe we could split this is in  :
should_returnProcessToTaskCandidateUser
should_returnProcessToTaskCandidateGroup